protected/humhub/modules/admin/models/forms/BasicSettingsForm.php

```diff
@@ -104,7 +102,9 @@ public function attributeHints()
     public function save()
     {
         Yii::$app->settings->set('name', $this->name);
-        Yii::$app->settings->set('baseUrl', $this->baseUrl);
+        if (!$this->isBaseUrlFixed()) {
```

Comment:
We don't need this condition. Like for `defaultTimeZone` which may be also fixed, the settings manager always returns the fixed value.